src/Auth/Azure.DataApiBuilder.Auth.csproj

```diff
@@ -8,14 +8,17 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <NoWarn>NU1603</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <WarningsNotAsErrors>NU1603</WarningsNotAsErrors>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <WarningsNotAsErrors>NU1603</WarningsNotAsErrors>
```

Comment:
What is needed to avoid getting this warning?

Reply:
Typically, the package sources do not contain the expected lower bound version. A higher version was used instead, which differs from what the package was authored against. To resolve this we would need to have that lower bound version.